ios/FluentUI/Bottom Sheet/CommandingItem.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// An object representing a command.
+///
+/// `CommandingItem` defines the high level properties and behavior of a command. Its visual representation is determined by
+/// the `BottomCommandingController`.
+@objc(MSFCommandingItem)
+open class CommandingItem: NSObject {
```

Comment:
Are these commands similar to those that might be in the CCB (Contextual Command Bar)? Is there opportunity for shared code there?

Reply:
Maybe in the future 😄 The API is designed with this in mind.